app/views/ladder/components/Leaderboard.js

```diff
@@ -64,6 +70,9 @@ module.exports = (LeaderboardView = (function () {
           { slug: 'when', col: 1, title: $.i18n.t('general.when') },
           { slug: 'fight', col: 1, title: '' }
         ]
+        if (this.hidesTeams) {
+          this.propsData.tableTitles.splice(5, 1)
```

Comment:
same here. otherwise 7 vs 5 maybe confused in the future

Reply:
we should not use magic-numbers like 5. Better to at least give them a variable name